packages/checkout/widgets-lib/src/components/Transactions/Transactions.tsx

```diff
@@ -144,66 +140,6 @@ export function Transactions({
     }
   }, [checkout, from]);
 
```

Comment:
I'm not sure why all of this is being deleted. This is handling WalletConnect differently to Passport and Metamask

Reply:
Talked it through